app/Item.php

```diff
@@ -578,7 +578,7 @@ public function freeFrom() {
         }
 
         if ($this->isAuthorUnknown()) {
-            return self::FREE_ALWAYS;
+            return self::UNKNOWN_AUTHOR_THRESHOLD;
```

Comment:
I would've just done 

```suggestion
            return 1940;
```

instead of defining a global constant at the top (encapsulation) but I see you're being consistent -- cool.